pages/signup.html

```diff
@@ -0,0 +1,57 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://litui.netlify.app/components/components-style.css">
+    <link rel="icon" href="./assets/sneakers.png" type="image/icon type">
+    <link rel="stylesheet" href="../styles/login-signup.css" />
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/latest/css/font-awesome.min.css" />
+    <link rel="stylesheet" href="../styles/utlis.css" />
+    <title>SneakerXclub</title>
+    
+</head>
+
+<body>
+    <div class="header">
+
+        <div class="bars">
+
+            <h2> <a class="logo" href="./index.html">SneakerXclub</a> </h2>
+        </div>
+
+
+        <div class="nav-icon">
+
+            <i class="fa fa-heart-o fa-2x p1" aria-hidden="true"></i>
+            <i class="fa fa-shopping-cart fa-2x p1" aria-hidden="true"></i>
+            <i class="fa fa-user-o fa-2x p1" aria-hidden="true"></i>
+        </div>
+    </div>
+    <div class="container">
+
+        <div id="signup-container" class="signup-container">
+            <form class="form input-form">
+                <div class="heading ">
+                    <p>Sign Up</p>
+                </div>
```

Comment:
instead of this use a h2 tag it will be better for SEO to, not a requirement but a good practice

Reply:
Sure , will do it.